### DISCUSSION TITLE
怎么用mcc完成自动私信在2b2t.xin宣传(How to use MCC to complete automatic private message in 2B2T. Xin propaganda)

### DISCUSSION THREAD

**Discussion by FIZxingchen** [ORelio/Minecraft-Console-Client#2553] [General]
del

**Comment by Black-HIG** [comment]
sending a comment and then delete it is not good for everyone
stop deleting your comments

**Comment by nannanthis** [comment]
Players' names should follow Minecraft's official naming rules
Using invalid characters (like CJK characters) can cause a lot of problems. For example, unable to join most of the offline servers or single player worlds in modern minecraft versions.

**Reply by Black-HIG** [reply]
You are A SB

**Comment by FIZxingchen** [comment]
ok,是我的错朋友(Okay, it's my fault, friend)
